opentelemetry-exporters-newrelic-auto/src/main/java/com/newrelic/telemetry/opentelemetry/export/auto/NewRelicSpanExporterFactory.java

```diff
@@ -0,0 +1,34 @@
+package com.newrelic.telemetry.opentelemetry.export.auto;
+
+import com.newrelic.telemetry.Attributes;
+import com.newrelic.telemetry.opentelemetry.export.NewRelicSpanExporter;
+import io.opentelemetry.sdk.contrib.auto.config.Config;
+import io.opentelemetry.sdk.contrib.auto.config.SpanExporterFactory;
+import io.opentelemetry.sdk.trace.export.SpanExporter;
+
+public class NewRelicSpanExporterFactory implements SpanExporterFactory {
+
+  private static final String NEWRELIC_API_KEY = "newrelic.api.key";
+  private static final String NEWRELIC_SERVICE_NAME = "newrelic.service.name";
+  private static final String DEFAULT_NEWRELIC_SERVICE_NAME = "(unknown service)";
+
+  /**
+   * Creates an instance of a {@link SpanExporter} based on the provided configuration.
+   *
+   * @param config The configuration
+   * @return An implementation of a {@link SpanExporter}
+   */
+  @Override
+  public SpanExporter fromConfig(Config config) {
+    final String apiKey = config.getString(NEWRELIC_API_KEY, "");
+    // todo: newrelic.service.name key will not required once service.name is provided via Resource
+    // in the SDK
+    final String serviceName =
+        config.getString(NEWRELIC_SERVICE_NAME, DEFAULT_NEWRELIC_SERVICE_NAME);
+
+    return NewRelicSpanExporter.newBuilder()
+        .commonAttributes(new Attributes().put("service.name", serviceName))
+        .apiKey(apiKey)
+        .build();
```

Comment:
I just remembered. We need to have support for the `uriOverride` option, and the `auditLoggingEnabled` option on this. Can you add those two options in here, as well?  If the Config doesn't have the options, then you shouldn't call the builder methods for them.

Reply:
awesome, thanks!